powerline/segments/common/bat.py

```diff
@@ -220,7 +208,7 @@ def battery(pl, format='{capacity:3.0%}', steps=5, gamify=False, full_heart='O',
 		})
```

Comment:
If `gamify` is true then `online` or `offline` symbol should be prepended with `['battery_online', 'battery_ac_state', 'battery_gradient', 'battery']` or `['battery_offline', 'battery_ac_state', 'battery_gradient', 'battery']` highlighting.